packages/react-native-cli/src/lib/Gradle.ts

```diff
@@ -123,18 +123,15 @@ export async function enableReactNativeMappings (
   try {
     const fileContents = await fs.readFile(appBuildGradlePath, 'utf8')
 
-    // If the file contains a 'bugsnag' configuration section already, add the
-    // 'uploadReactNativeMappings' flag to it
-    if (/^\s*bugsnag {/m.test(fileContents)) {
-      await insertValueAfterPattern(
-        appBuildGradlePath,
-        /^\s*bugsnag {[^}]*?(?=})/m,
-        '  uploadReactNativeMappings = true\n',
-        ENABLE_REACT_NATIVE_MAPPINGS_REGEX,
-        logger
+    if (/^\s*uploadReactNativeMappings\s*=\s*true/m.test(fileContents)) {
```

Comment:
Remove references to uploadMappings, etc, in function/param names.
Remove the ENOENT catch and pattern not found checks

1. Read Gradle file
2. Check for uploadReactNativeMappings and warn if found
3. If uploadEndpoint or buildEndpoint are defined, insert `bugsnag {}` section